api/src/Microsoft.Azure.IIoT.OpcUa.Api.Publisher/src/Models/GetConfiguredNodesOnEndpointsRequestApiModel.cs

```diff
@@ -0,0 +1,56 @@
+﻿// ------------------------------------------------------------
```

Comment:
I think we don't really need this model. we already have the PublishNodesRequestApiModel that covers all the content here.

Reply:
removed. Using PublishNodesRequestApiModel